samples/cliConfig.json

```diff
@@ -2,13 +2,17 @@
   "label": "AFJ Rest Agent",
   "walletId": "walletId",
   "walletKey": "walletKey",
+  "postgresHost": "alice-agent-postgres:5432",
```

Comment:
to be picky this isn't a host, this is a host and a port. I'd consider splitting them into separate config items